app/assets/stylesheets/elements/_alert.scss

```diff
@@ -1,28 +1,46 @@
 @keyframes move-in-alert {
   from {
-    transform: translateY(200%);
+    transform: translateY(100vh);
   }
 
   to {
     transform: translateY(0);
   }
 }
 
-.alert {
+.alerts {
   position: fixed;
   right: $margin / 3;
   bottom: $margin / 3;
+  display: flex;
+  flex-direction: column;
+  align-items: end;
+  gap: $margin / 3;
+  width: min(35em, 75%);
+  z-index: 2000;
+}
+
+.alerts__alert {
   padding: .5em 1em;
+  display: flex;
+  align-items: center;
+  gap: 1em;
   border-radius: $border-radius;
   background: $primary;
   box-shadow: 0 10px 10px rgba($black, .5);
   color: $pure-white;
   font-size: 14px;
   text-decoration: none;
-  z-index: 2000;
-  transform: translateY(200%);
+  transform: translateY(100vh);
   animation: move-in-alert 200ms 500ms forwards;
-  max-width: min(50em, 75%);
+
+  button {
+    background-color: transparent;
+  }
+
+  p {
+    max-width: 65ch;
```

Comment:
Might be better to not use `ch` since we don't use it anywhere else

Reply:
Ah, then I think `em` will probably suffice here.